python-sdk/conftest.py

```diff
@@ -294,6 +296,16 @@ def _upload_or_delete_remote_file(file_create, object_prefix, provider, source_p
             hook.load_file(source_path, object_prefix, bucket_name)
         else:
             hook.delete_objects(bucket_name, object_prefix)
+    elif provider == "azure":
+        from airflow.providers.microsoft.azure.hooks.wasb import WasbHook
+
+        bucket_name = os.getenv("AZURE_BUCKET", "astro-sdk")
+        object_path = f"wasb://{bucket_name}/{object_prefix}"
```

Comment:
we need to provide `abfss` url as well because wasb is going to be deprecated.

https://learn.microsoft.com/en-us/azure/databricks/external-data/wasb-blob